modules/RTC/TraceablePeerConnection.js

```diff
@@ -295,20 +284,7 @@ export default function TraceablePeerConnection(
 
     this.interop = new Interop();
```

Comment:
Yes, we need it still for converting the SDP into the format that Jicofo expects in all source signaling which is still in plan-b format. Plan is to bring it into the library and rename it appropriately instead of calling it interop in a follow up PR.

Reply:
👍